core/templates/dev/head/admin/admin.html

```diff
@@ -11,93 +11,217 @@
         DEMO_EXPLORATION_IDS: JSON.parse('{{demo_exploration_ids|js_string}}')
       }
     </script>
+    <style>
+    html, body {
+      background-color: #eee;
+    }
+    </style>
   </head>
 
   <body ng-controller="Admin" ng-cloak>
-    <div>
-      Click <a href="/gallery">here</a> to go to the gallery.
-    </div>
-    <br>
-    <div ng-if="message" style="background-color: lightyellow; position: fixed; width: 30%; z-index: 3000; border: 1px solid brown; right: 5px;">
+    <nav class="navbar navbar-default oppia-navbar oppia-prevent-selection" role="navigation">
+      <div class="navbar-container">
+        <div class="navbar-header protractor-test-navbar-header pull-left">
+          <a class="oppia-navbar-brand-name oppia-transition-200" href="/gallery" focus-on="<[LABEL_FOR_CLEARING_FOCUS]>">
```

Comment:
Please go through this carefully and remove irrelevant stuff -- e.g. I don't think LABEL_FOR_CLEARING_FOCUS or windowIsNarrow are defined on the admin page's scope.